src/js/components/ToggleCheckboxList/ToggleCheckboxList.tsx

```diff
@@ -0,0 +1,207 @@
+import React, { createRef, FunctionComponent, useState } from 'react';
+import { Panel } from '../Panel';
+import { PanelHeader } from '../PanelHeader';
+import { PanelBody } from '../PanelBody';
+import { PanelRow } from '../PanelRow';
+import { Tag } from '../Tag';
+import { Field } from '../Field';
+import { ListToggle } from '../ListToggle';
+
+type Props = {};
+
+const ToggleCheckboxList:FunctionComponent<Props> = () => {
+  const [activeRows, setActiveRows] = useState([]);
+  const [headerStatus, setHeaderStatus] = useState({});
+
+  const programData = [{
```

Comment:
nit: I know you have plans to clean up after the data names are more final, but let's not define this here. Maybe for now define this test data in the story and pass it in as a prop.